patches/arcade/0012-Disable-ILRewrite-step-this-isn-t-built-in-source-bu.patch

```diff
@@ -0,0 +1,26 @@
+From 893bccaed963504f49bd186f80f70b43d748a2ef Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Sun, 16 Aug 2020 18:29:04 -0500
+Subject: [PATCH 12/12] Disable ILRewrite step - this isn't built in
+ source-build so it fails.
```

Comment:
Any idea what ILRewrite does, is our SDK defective? Issue tracking?

Reply:
Added an issue https://github.com/dotnet/source-build/issues/1728.  Note that this is just preserving the status quo - we've never built ILRewrite as far as I can tell but previously this target didn't get called.  Now that it was being called, we were failing without having those tasks available.